### ISSUE TITLE
Add Plot Fiber 3D / Animate fiber 3D

### ISSUE THREAD

**Issue by cslotboom** [u-anurag/OpenSeesPy#65]
There should be a procedure added to that can plot and animate fibers in 3D models.

**Comment by u-anurag** [comment]
Does it need to be in this release?

**Comment by cslotboom** [comment]
No it doesn't, it could be in V3 or later.

**Comment by u-anurag** [comment]
Is this function ready to go or you need a few days to implement this?

**Comment by cslotboom** [comment]
No, sorry, this is not good to go. In 3D I'm still not sure how to record the data properly, so it would take awhile to figure out. 

I just thought it would be a potential feature for future releases. Many people will have 3D fiber sections they need to analyze.

**Comment by u-anurag** [comment]
Sure. Version 3 then.
